File: spring-multirabbit-examples/spring-multirabbit-example-java/src/main/java/com/freenow/multirabbit/example/java/Application.java
package com.freenow.multirabbit.example.java;

import org.springframework.amqp.rabbit.annotation.EnableRabbit;
import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;

@EnableRabbit
@SpringBootApplication
public class Application {

    public static void main(final String... args) {
        SpringApplication.run(Application.class, args);
    }
}

File: spring-multirabbit-examples/spring-multirabbit-example-kotlin/src/main/kotlin/com/freenow/multirabbit/example/Application.kt
package com.freenow.multirabbit.example

import org.springframework.amqp.rabbit.annotation.EnableRabbit
import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.SpringBootApplication

fun main(args: Array<String>) {
    SpringApplication.run(Application::class.java, *args)
}

@EnableRabbit
@SpringBootApplication
class Application
